packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -527,18 +499,81 @@ const modalScreenOptions: ScreenOptions = Platform.select({
   }),
 })
 
+const animatedScreens = (Navigator: typeof Stack) => (
+  <>
+    <Navigator.Screen name={Screens.Send} component={Send} options={sendScreenNavOptions} />
+    <Navigator.Screen
+      name={Screens.Language}
+      component={Language}
+      options={Language.navigationOptions}
+    />
+    <Navigator.Screen
+      options={headerWithBackButton}
+      name={Screens.SelectLocalCurrency}
+      component={SelectLocalCurrency}
+    />
+    <Navigator.Screen
+      name={Screens.PincodeEnter}
+      component={PincodeEnter}
+      options={{
+        ...headerWithBackButton,
+        gestureEnabled: false,
+        cardStyleInterpolator: ({ current }) => ({
+          containerStyle: {
+            opacity: current.progress,
+          },
+        }),
+      }}
+    />
```

Comment:
Screen options should be moved to the component `navigationOptions` field.

Reply:
Moved the options to the respective files :)